src/TemplateChooser.php

```diff
@@ -32,7 +32,9 @@ public function homepage($content)
         $templates = [];
 
         // First candidate: Theme-specific config.yml file.
-        $templates[] = $this->config->get('theme/homepage_template');
+        if ($this->config->get('theme/homepage_template')) {
```

Comment:
I know, but I made it in line with the dozen or so other similar statements in that file. If we change this one, we should change all of them to keep it consistent.